assets/app/view/game/round/merger.rb

```diff
@@ -54,6 +54,7 @@ def render
           buttons << render_offer(entity, auctioning_corporation) if actions.include?('assign')
 
           buttons << render_merge(entity) if actions.include?('merge')
+          buttons << h(ScrapTrains, corporation: entity) if actions.include?('scrap_train')
```

Comment:
why isn't this just done where discard trains is added? game_page.rb

Reply:
Maybe. It's a bit tricky in that this is always an available (if you have a pullman) optional action, much like "take loan". In my opinion optional actions, should be at the bottom of the displayed buttons to minimize the attention they draw. They are there if you want them, but not demanding your attention.

Reply:
I was thinking similarly to crericha, since the pullman scrapping is an optional action that can be done nearly any time like taking a loan I thought it would be best to keep the button near that